test/controllers/area.controller.test.js

```diff
@@ -319,4 +322,322 @@ describe('Area Controller', () => {
       });
     });
   });
+
+  describe('Update Area', () => {
+    const areaId = mongoose.Types.ObjectId();
+    const area = AreaFactory.build({
+      _id: areaId,
+      area: 'ibadan',
+      state: 'oyo',
+      longitude: 40.7128,
+      latitude: 74.006,
+    });
+    const updatedArea = {
+      id: areaId,
+      area: 'oshogbo',
```

Comment:
I am still concerned about writing this in a normal case. The truth is that this will mostly come as a normal case from the UI, so it makes sense to simulate the real-world usage in your tests (by making this a normal case) to easily catch any error from using different case.

Don't forget, the main purpose of writing tests is to catch bugs as early as possible when the application is used.

```suggestion
      area: 'Oshogbo',
```